packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -866,6 +880,11 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return this.context.attachState;
     }
 
+    public get idCompressor(): IdCompressor | undefined {
```

Comment:
Do we need this getter? Can the property just be made public instead (since it is readonly)?

Reply:
You're right, not sure why this was a getter. Removed in favor of a `public readonly` property.